src/ext/FirewallExtension/wixlib/pl-pl.wxl

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  <copyright file="pl-pl.wxl" company="Outercurve Foundation">
+    Copyright (c) 2004, Outercurve Foundation.
+    This software is released under Microsoft Reciprocal License (MS-RL).
+    The license and further copyright text can be found in the file
+    LICENSE.TXT at the root directory of the distribution.
+  </copyright>
+-->
+<WixLocalization Culture="pl-pl" xmlns="http://wixtoolset.org/schemas/v4/wxl">
+    <String Id="msierrFirewallCannotConnect" Overridable="yes">Nie uda�o si� po��czy� z Zapor� systemu Windows.  ([2]   [3]   [4]   [5])</String>
```

Comment:
These characters look like the ones you fixed in your last commit for the SqlExtension translation.  Can you fix these, too?